lib/Server.js

```diff
@@ -91,6 +91,7 @@ class Server {
     // TODO this.<property> is deprecated (remove them in next major release.) in favor this.options.<property>
     this.hot = this.options.hot || this.options.hotOnly;
     this.headers = this.options.headers;
+    this.profile = !!this.options.profile;
```

Comment:
`this.options.profile`, don't use `this.<option>` it is unnecessary and all `this.<option>` will be removed in next major release